druntime/src/importc.h

```diff
@@ -152,6 +152,10 @@
 #define __PRETTY_FUNCTION__ __func__
 
 #ifndef __aarch64__
+#define _Float32 float
+#define _Float32x float
```

Comment:
Does nobody read the spec? 

C23 - H.2.3 Extended floating types
> For each of its basic formats, IEC 60559 specifies an extended format whose maximum exponent and
precision exceed those of the basic format it is associated with. Extended formats are intended for
arithmetic with more precision and exponent range than is available in the basic formats used for
the input data. The extra precision and range often mitigate round-off error and eliminate overflow
and underflow in intermediate computations. The table below gives the minimum values of these
parameters, as defined for the C floating-point model (5.2.4.2.2). For all IEC 60559 extended (and
interchange) formats, emin = 3 emax .

Simply put, `x` is the next precision up, i.e: `alias _Float32x = double;`